css/default.css

```diff
@@ -61,7 +59,6 @@ a:hover {
 
 #content {
   padding-top: 1em;
-  border-top: 1px dotted #808000;
```

Comment:
What I mean to say is, it's not hurting anything but it could help if you want a certain type of aesthetic.